src/iconsManifest/supportedExtensions.ts

```diff
@@ -406,6 +406,22 @@ export const extensions: IFileCollection = {
       languages: [languages.apib],
       format: FileFormat.svg,
     },
+    {
+      icon: 'apl',
+      extensions: [
```

Comment:
You don't need to specify the extensions as we're relying on the language id, which is already supporting these extensions.

Reply:
Argh, I always forget that thing x.x! I will fix it as soon as possible.